go/vt/vtgate/planbuilder/testdata/dml_cases.json

```diff
@@ -6199,5 +6199,28 @@
         "user.user"
       ]
     }
+  },
+  {
+    "comment": "insert into ref; TODO(maxeng) is this a bug?",
```

Comment:
Tested this on `main` branch and get the same results. Is this a bug? When we do similar `UPDATE` or `DELETE` we get a `primary vindex not found` error, but this `INSERT` succeeds.

Reply:
issue for a different PR

Reply:
👍 want me to revert this test case or leave in place in this PR?

Reply:
Leaving as-is is good